File: 2575.py
# Gripe-143
# https://jv.umsa.bo/problem.php?id=2575

n, k = map(int, input().split())
nums = list(map(int, input().split()))
suma = 0
for i in range(n-k+1):
    suma += sum(nums[i:i+k])/k
    # print(suma)
print(int(suma))

File: jvCPP/2575.cpp
// Gripe-143
// https://jv.umsa.bo/problem.php?id=2575

#include "iostream"

using namespace std;
int main(){
    int n, k;
    cin >> n >> k;
    int nums[n];
    for (int i = 0; i < n; i++){
        cin >> nums[i];
    }

    double suma = 0.0;
    for (int i = 0; i < n-k+1; i++){
        int s = 0.0;
        for (int j = i; j < k+i; j++){
            s += nums[j];
        }
        suma += double(s)/double(k);
    }
    cout << int(suma) << endl;
    return 0;
}